service-front/module/Application/src/Services/SiriusApiService.php

```diff
@@ -99,11 +113,25 @@ public function checkAuth(RequestInterface $request): bool
      */
     public function getLpaByUid(string $uid, Request $request): array
     {
+        $preferHeader = in_array($uid, $this->preferLpas) ? // we need this to test LPA responses
+            ['Prefer' => sprintf("example=%s", $uid)] : // only while using the sirius mock
+            [];
```

Comment:
I don't feel very comfortable about adding code related to mocking things locally to our production codebase.

Reply:
It would need to be removed once the sirius mock is replaced with the actual service. It's needed before that, otherwise there's no way to use the system unless the sirius mock returns behaviourally reliable data